gensim/matutils.py

```diff
@@ -471,9 +471,8 @@ def jaccard(vec1,vec2):
         intersection = 0
         union = sum(weight for id_, weight in vec1) + sum(weight for id_, weight in vec2)
         vec1, vec2 = dict(vec1), dict(vec2)
-        for item in vec1:
-            if item in vec2:
-                intersection = min(vec2[item], vec1[item]) + intersection
+        for feature_id, feature_weight in iteritems(vec1):
```

Comment:
Put closer to the `intersection = 0.0` initialization. They belong together conceptually, so should be close together visually too.